packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -748,6 +751,11 @@ export default class NewVm extends BaseComponent {
     template => template && template.virtualizationMode === 'hvm'
   )
 
+  _templateNeedVtpm = createSelector(
+    () => this.props.template,
+    template => template && Boolean(template.platform?.vtpm)
```

Comment:
Let's be more explicit, if that's actually the condition you want to test:
```suggestion
    template => template?.platform?.vtpm === true
```